check_circular_redirects.py

```diff
@@ -0,0 +1,35 @@
+# This script checks for circular 301 redirects in the config/redirects
```

Comment:
Question:
Did you mean to add this to the PR? I think it's probably better to leave anything not essential for building the docs out of docs repos.

Reply:
I did not. Thanks for the catch.